packages/fast-jss-manager-react/src/manage-jss.spec.tsx

```diff
@@ -33,6 +33,17 @@ const dynamicStyles: ComponentStyles<any, any> = {
     }
 };
 
+const stylesheetResolver: ComponentStyles<any, any> = (config: any): any => {
+    return {
+        resolvedStylesClass: {
+            background: "green",
+            color: (): string => {
+                return "yellow";
+            }
+        }
+    };
+};
+
 const staticAndDynamicStyles: ComponentStyles<any, any> = {
```

Comment:
comments would be great here.  what does this do?

Reply:
fixed.